docs/reference/rotationalmotor.md

```diff
@@ -23,6 +23,9 @@ It defaults to `"rotational motor"`.
 - The `maxTorque` field specifies both the upper limit and the default value for the motor *available torque*.
 The *available torque* is the torque that is available to the motor to perform the requested motions.
 It is expressed in *newton meter* [N⋅m].
+When used in velocity mode, this maximum torque is always applied (with the correct sign) until the speed is reached. 
+See : http://ode.org/wiki/index.php?title=Manual#Stops_and_motor_parameters
```

Comment:
```suggestion
- The `maxTorque` field specifies both the upper limit and the default value for the motor *available torque* and is expressed in *newton meter* [N⋅m].
The *available torque* is the torque that is available to the motor to perform the requested motions.
When used in velocity control, this maximum torque is always applied (with the correct sign) until the target speed is reached. More details are available [here](http://ode.org/wiki/index.php?title=Manual#Stops_and_motor_parameters).
```

Reply:
It's totally fine, completely forgot to use markdown formatting !